<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_SpecViewer3</class>
 <widget class="QWidget" name="Form_SpecViewer3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QScrollBar" name="verticalScrollBar">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Form_SpecViewer1" name="panel1" native="true"/>
   </item>
   <item>
    <widget class="Form_SpecViewer1" name="panel2" native="true"/>
   </item>
   <item>
    <widget class="Form_SpecViewer1" name="panel3" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Form_SpecViewer1</class>
   <extends>QWidget</extends>
   <header>include/form_specviewer1.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
